test/e2e/storage/vsphere/vsphere_volume_diskformat.go

```diff
@@ -73,15 +73,15 @@ var _ = utils.SIGDescribe("Volume Disk Format [Feature:vsphere]", func() {
 		if !isNodeLabeled {
 			nodeLabelValue = "vsphere_e2e_" + string(uuid.NewUUID())
 			nodeKeyValueLabel = make(map[string]string)
-			nodeKeyValueLabel["vsphere_e2e_label"] = nodeLabelValue
-			framework.AddOrUpdateLabelOnNode(client, nodeName, "vsphere_e2e_label", nodeLabelValue)
+			nodeKeyValueLabel["vsphere_e2e_label_volume_diskformat"] = nodeLabelValue
```

Comment:
You could also define these as constants to reduce any future typo mistakes.

Reply:
@msau42 Addressed your review comments. used `NodeLabelKey` const to avoid typos.